samples/peripheral/radio_test/src/radio_test.c

```diff
@@ -211,7 +216,6 @@ static void radio_config(nrf_radio_mode_t mode, enum transmit_pattern pattern)
 		 * 10-bit preamble.
```

Comment:
Sorry for commenting a non-changed line, but this should be fixed: the preamble lentgh is 10 **bytes**